pages/posts/flight-of-a-byte.tsx

```diff
@@ -0,0 +1,112 @@
+                </p>
+                <p>
+                When the Wesh app on the computer for <code>client2</code> sends the bytes of a message, what is the <code className="language-go">“flight path”</code> of the bytes of <code className="language-go">“Hello”</code> as they travel to the computer of <code>client1</code>? It&apos;s different from “traditional” networking where, for example, a computer sends the message in a UDP packet to the IP address of the other computer, as described in this Wikipedia article. Instead of a specific IP address, Wesh communication happens between libp2p peers, where the client&apos;s PeerID is the value that we&apos;ve seen in previous examples and is the same no matter how the client is connected to the network.
+                </p>
+                <h3>Peer discovery</h3>
+                <p>
+                The “glue” between a libp2p peers and a more traditional network address is <a href="https://docs.libp2p.io/concepts/discovery-routing/overview/">peer discovery</a>. There are a lot of details that we won&apos;t go into regarding types of discovery, peer authentication, etc. In this blog post, we just describe the minimal steps for how <code>client1</code> receives the message from <code>client2</code>. When the libp2p service of client1 discovers the computer of <code>client2</code>, it opens a network connection (<a href="https://www.notion.so/Backlogs-for-1-Berty-Messenger-Launch-and-2-Wesh-Network-d420a66eea244942878b31d3fc6eab4a?pvs=21">code</a>) and reads from it using a traditional byte stream (<a href="https://github.com/libp2p/go-libp2p-pubsub/blob/6d73cd4b56bf9d7fdd0c0caaed629a3d51027a75/comm.go#L66">code</a>). (We give links to code for curious readers, but this isn&apos;t a code tutorial and you can skip the code links.)
+
+                We&apos;ve mentioned the traditional networking code “under the hood” to remove some of the mystery. But that code doesn&apos;t immediately read the message from <code>client2</code>. Instead, there are some communication steps which allow Wesh to use libp2p to operate in a peer-to-peer environment.
+                </p>
+                
+                <h3>Message log and “head”</h3>
+                <p>
+                To send a message in traditional networking, one user connects to the network address of the other user and transmits the message. Or the user connects to a central server and appends to the message log there, while the other user connects to the same central server and reads the message log. But this doesn&apos;t work in a peer-to-peer environment. Instead, the goal is for each group member to have a copy of the message log, where there is a “head” (the latest message) which refers to all the previous messages in a chain. To send a message to the group, a member adds to the local copy of the message log and updates the “head”. This will be synchronized with the other members.
+                </p> 
+                <p>
```

Comment:
This is missing a paragraph. Please include the paragraph from the Notion page starting with "Without a central server..."